### ISSUE TITLE
Please add Intepreter compiler

### ISSUE THREAD

**Issue by zminhquanz** [jpd002/Play-#313]
please add interpreter compiler , because it's useful to debug register and graphic issue

**Comment by jpd002** [comment]
Indeed, it's quite a pain to debug ATM. I've been thinking of adding this for quite a long time but never took time to actually do it.
